modules/installation-aws-user-infra-rhcos-ami.adoc

```diff
@@ -18,48 +18,51 @@ You must use a valid {op-system-first} AMI for your Amazon Web Services
 |AWS AMI
 
 |`ap-northeast-1`
-|`ami-0c63b39219b8123e5`
+|`ami-03d80bc757baa3ea3`
 
 |`ap-northeast-2`
-|`ami-073cba0913d2250a4`
+|`ami-0d38ddacba83423c9`
 
 |`ap-south-1`
-|`ami-0270be11430101040`
+|`ami-0167e201cc0375d94`
 
 |`ap-southeast-1`
-|`ami-06eb9d35ede4f08a3`
+|`ami-09e527c00c9470b5f`
 
 |`ap-southeast-2`
-|`ami-0d980796ce258b5d5`
+|`ami-0936f38cd6161ad80`
 
 |`ca-central-1`
-|`ami-0f907257d1686e3f7`
+|`ami-083a6c4d5fec9e7f7`
 
 |`eu-central-1`
-|`ami-02fdd627029c0055b`
+|`ami-00731d73fad047d6a`
+
+|`eu-north-1`
+|`ami-0359348d6d4ede973`
```

Comment:
RHCOS is here, but the installer doesn't support it yet: openshift/installer#2308.  Probably leave it off until we get that landed, since it's unlikely to get backported to 4.2, and leaving it off now will make it easier to backport docs to 4.2 (although the master AMIs have diverged from the 4.2 AMIs; looks like you're using the master AMIs here).